Cannot find module 'openapi-types' or its corresponding type declarations.

## 🐛 Bug Report

When using typescript the following error occurs with version 4.0.0 (because of #333):

```
node_modules/fastify-swagger/index.d.ts(2,47): error TS2307: Cannot find module 'openapi-types' or its corresponding type declarations.
```

## To Reproduce

Use typescript

## Expected behavior

No error

## Your Environment

- *node version*: 15
- *fastify version*: latest
- *os*: Mac

--- a/package.json
+++ b/package.json
@@ -36,7 +36,6 @@
     "fs-extra": "^9.0.0",
     "joi": "^14.3.1",
     "joi-to-json-schema": "^5.1.0",
-    "openapi-types": "^7.0.1",
     "pre-commit": "^1.2.2",
     "standard": "^16.0.1",
     "swagger-parser": "^10.0.2",
@@ -49,7 +48,8 @@
     "fastify-plugin": "^3.0.0",
     "fastify-static": "^3.3.0",
     "js-yaml": "^4.0.0",
-    "json-schema-resolver": "^1.2.0"
+    "json-schema-resolver": "^1.2.0",
+    "openapi-types": "^7.0.1"
   },
   "standard": {
     "ignore": [

